### ISSUE TITLE
training with 4gpus with non-local 

### ISSUE THREAD

**Issue by gooners1886** [facebookresearch/video-nonlocal-net#20]
I have trained a model using run_i3d_baseline_300k_4gpu.sh on the kinetics dataset according to the guide in ReadMe.md.  I get same top-1 and top-5 accuracy in the README.md.  I want to train a i3d model **with non-local** using 4gpus. But there is no top-1 and top-5 accuracy result in the ReadMe.md file. Is this possible for me to  train this kind of model and getting higher accuracy than baseline?  Why this model is not released?

**Comment by xiaolonw** [comment]
yes, of course you will get improvement over baseline. 
We have not released too many models for avoiding confusions. 
The performance in this case will be around 1% less than run_i3d_nlnet_400k.sh (74.7/91.6) because the iteration number is much less, 300k in 4GPUs versus 400k in 8GPUs.

**Comment by gooners1886** [comment]
@xiaolonw  thank you very much  and one more question: 

Is this the right script to conduct my comparison  experiment between   baseline and non-local using 4gpus?

CHECKPOINT_DIR=../data/checkpoints/run_i3d_baseline_300k_4gpu
mkdir ${CHECKPOINT_DIR}

python ../tools/train_net_video.py \
--config_file ../configs/DBG_kinetics_resnet_4gpu_c2d_nonlocal_300k.yaml \
TRAIN.PARAMS_FILE ../data/pretrained_model/r50_pretrain_c2_model_iter450450_clean.pkl \
VIDEO_DECODER_THREADS 5 \
NONLOCAL.CONV3_NONLOCAL **True** \
NONLOCAL.CONV4_NONLOCAL **True** \
MODEL.VIDEO_ARC_CHOICE 2 \
TRAIN.DROPOUT_RATE 0.0 \
CHECKPOINT.DIR ${CHECKPOINT_DIR} \
DATADIR ../data/lmdb/kinetics_lmdb_multicrop/ \
FILENAME_GT ../process_data/kinetics/vallist.txt \
2>&1 | tee ${CHECKPOINT_DIR}/log.txt

**Comment by xiaolonw** [comment]
correct
